packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -254,11 +255,84 @@ The requested scope is invalid:
+> **Note:** The `/device/authorize` endpoint requires client authentication. See the [Client authentication methods](#client-authentication-methods) section for more information on which method to choose and how to use the parameters in your request.
+
+| Parameter               | Description                                                                                                                                                                                                                                                                                                                        | Type   |
+| :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----- |
+| client_id               | Obtained during either manual client registration or through the [Dynamic Client Registration API](/docs/reference/api/oauth-clients/). It identifies the client and must match the value preregistered in Okta.                                                                                                                       | String |
+| scope                   | A list of scopes that the client wants included in the access token. See [scopes](#access-token-scopes-and-claims).                                                                                                                                                                                                 | String |
+
+#### Response properties
+
+Based on the scopes requested. Generally speaking, the scopes specified in a request are included in the access token in the response.
+
+| Property                  | Description                                                                                    | Type    |
+| :------------------------ | :--------------------------------------------------------------------------------------------- | :------ |
+| verification_uri          | The verification URI that the resource owner must visit to authenticate.                            | String  |
+| verification_uri_complete | The complete verification URI that the resource owner can alternatively visit to authenticate.      | String  |
+| device_code               | The device code to be kept hidden from the resource owner.                                     | String  |
+| user_code                 | The user code to be shown to the resource owner.                                               | String  |
+| interval                  | The interval time the client should wait until it can poll the /token endpoint in seconds.     | Integer |
+| expires_in                | The expiration time of the device code and user code in seconds.                               | Integer |
+
+#### List of errors
```

Comment:
Possible errors

Reply:
This heading name is common across the doc, we always call it `List of errors`. I'll leave it as it is, otherwise it's going to look inconsistent with the rest of the docs.

Reply:
Ha, actually we use `Possible errors` for authorize and `List of errors` for tokens  @susanharper-okta which one we should use?

Reply:
@larsjohansen-okta and @yuliu-okta - that is not currently defined in our [API Reference Template](https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/748062101/REST+API+Reference+Markdown+Template) that we provide for Devs to use to create new API Reference docs. However, I like **Possible errors** better than **List of errors**. I feel that Possible errors is more descriptive than just saying "List". Let's go with Possible errors and I'll add it to the above mentioned template. Thanks.